src/components/core/menu/mainMenu/MainMenuRightBurger/useBurgerMenuItems.tsx

```diff
@@ -133,7 +118,7 @@ export const useBurgerMenuItems = () => {
               <span>{currency}</span>
               <span className="flex items-center">
                 <IconV
-                  className={`invisible h-12 w-12 me-8 ${
+                  className={`invisible h-12 w-12 ${
```

Comment:
just making sure this margin was removed on purpose? I assume it was added because of RTL support?

Reply:
Removed on purpose as it was added by the #658 PR for RTL support